lib/menu.rb

```diff
@@ -0,0 +1,44 @@
+class Menu
+  attr_accessor :list_of_dishes
+
+  def initialize
+    @list_of_dishes = []
+  end
+
+  def add_to_menu(dish)
+    @list_of_dishes.push(dish.itemise)
+  end
+
+  def access_menu
+    @list_of_dishes = DEFAULT_MENU
```

Comment:
When I'm reading this method, it feels like it's "resetting" the list of dishes to a default one — so maybe a better name for it could be "reset_menu" or "initialize_default_menu"?